falcon/testing/client.py

```diff
@@ -282,6 +283,10 @@ def simulate_request(app, method='GET', path='/', query_string=None,
                 Accepts both byte strings and Unicode strings
                 (default: ``None``). If a Unicode string is provided,
                 it will be encoded as UTF-8 in the request.
+            json(JSON serializable): A JSON document to send as the body
```

Comment:
It may be helpful to replace "send" with "serialize" to reinforce to the reader that they are expected to pass a dict, list, etc. rather than the already-serialized JSON string.